index.html

```diff
@@ -1692,8 +1688,93 @@ <h5>Implementations</h5>
         </aside>
         <img src="img/room_perspective.png" />
 
+        <p>To establish a tunnel, one of the peers (the "source") calls the RPC <em>tunnel.connect</em> on the portal peer, specifying the "target" peer's ID:</p>
+        <figure class="request-response">
+            <div class="request">
+                <div class="header">
+                    <span class="key">Request number</span><span class="value">1</span>
+                    <span class="key">Body type</span><span class="value">JSON</span>
+                    <span class="key">Stream</span><span class="value">Yes</span>
+                    <span class="key">End/err</span><span class="value">No</span>
+                </div>
+                <pre><code>{
+  "name": ["tunnel", "connect"],
+  "type": "duplex",
```

Comment:
According to the earlier description of the RPC protocol: "For each procedure in the RPC protocol you must already know whether it is source or async", this additonal "duplex" type should be introduced in #rpc-protocol